src/state_machine.zig

```diff
@@ -267,6 +272,20 @@ pub const StateMachine = struct {
         return results_count * @sizeOf(Account);
     }
 
+    fn execute_lookup_transfers(self: *StateMachine, input: []const u8, output: []u8) usize {
+        const batch = std.mem.bytesAsSlice(u128, input);
+        var output_len = @divFloor(output.len, @sizeOf(Transfer)) * @sizeOf(Transfer);
+        var results = std.mem.bytesAsSlice(Transfer, output[0..output_len]);
```

Comment:
Ah, this was my bad, from early days, using `var` instead of `const`.

Would you like to make both of these `var output_len` and `var results` declarations `const` instead of `var`? Same for `execute_lookup_accounts()` above?

Reply:
Fixed. 🎉